src/harness/testcases/configs/test_WINNF_FT_S_SCS_17/default.config

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't think this is used anymore.

Reply:
This   file has been removed as the associated test case has been removed.